packages/web3torrent/src/utils/checkTorrentInTracker.ts

```diff
@@ -1,33 +1,33 @@
 import {Client} from 'bittorrent-tracker';
 import {defaultTrackerOpts} from '../constants';
 import {TorrentTestResult} from '../library/types';
+import debug from 'debug';
```

Comment:
The logging will need to change when https://github.com/statechannels/monorepo/pull/1473/files is merged.

Reply:
well... it's not merged.
As this log is not essential at all, I would merge this PR, so that the CI error stops appearing, and later change the logging